### DISCUSSION TITLE
Not able to proceed my Student developer pack application

### DISCUSSION THREAD

**Discussion by khushinim** [community/community#132823] [GitHub Education]
### Hi everyone,

I am not able to re-verify my education status, when I click proceed my application after uploading proof of my education, I'm again prompted to that proof uploading screen with no data which I had previously uploaded, due to this I'm not able to proceed my application, I have tried several times but the same thing happens again and again.
Kindly resolve my issue as soon as possible.

**Comment by SamuelOUS** [comment]
Hi Khushinim!

I had the same issue. You should give the page some time to refresh and then try again. If this doesn't help, try doing it in an incognito window. It worked for me :)

If you have any other questions or need further assistance, feel free to ask. I'm here to help!
